backend/de.metas.business/src/main/java/de/metas/invoice/detail/InvoiceDetailCloneMapper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.invoice.detail;
+
+import de.metas.invoice.InvoiceId;
+import de.metas.invoice.InvoiceLineId;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import org.adempiere.exceptions.AdempiereException;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+
+@Builder
+public class InvoiceDetailCloneMapper
+{
+	@NonNull @Getter private final InvoiceId originalInvoiceId;
+	@NonNull @Getter private final InvoiceId targetInvoiceId;
+	@Nullable private final ClonedInvoiceLinesInfo clonedInvoiceLinesInfo;
```

Comment:
IMHO this shall be non null, because there is no point in being null.

Reply:
made it non null